api/soap/mantisconnect.wsdl

```diff
@@ -259,6 +266,43 @@
    <xsd:element name="url" type="xsd:string" minOccurs="0"/>
   </xsd:all>
  </xsd:complexType>
+ <xsd:complexType name="FilterSearchData">
+  <xsd:all>
+   <xsd:element name="project_id" type="tns:IntegerArray" minOccurs="1"/>
+   <xsd:element name="search" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="category" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="severity_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="status_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="priority_id" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="reporter_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="handler_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="note_user_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="resolution_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="product_version" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="user_monitor_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="hide_status_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="sort" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="sort_direction" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="sticky" type="xsd:boolean" minOccurs="0"/>
+   <xsd:element name="view_state_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="fixed_in_version" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="target_version" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="platform" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="os" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="os_build" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="start_day" type="xsd:integer" minOccurs="0"/>
+   <xsd:element name="start_month" type="xsd:integer" minOccurs="0"/>
+   <xsd:element name="start_year" type="xsd:integer" minOccurs="0"/>
+   <xsd:element name="end_day" type="xsd:integer" minOccurs="0"/>
+   <xsd:element name="end_month" type="xsd:integer" minOccurs="0"/>
+   <xsd:element name="end_year" type="xsd:integer" minOccurs="0"/>
+   <xsd:element name="tag_string" type="tns:StringArray" minOccurs="0"/>
+   <xsd:element name="tag_select" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="custom_fields" type="tns:FilterCustomFieldArray" minOccurs="0"/>
+   <xsd:element name="page_number" type="xsd:integer" minOccurs="1"/>
```

Comment:
Technically speaking the page_number and issues_per_page are not part of the filter definition, but are part of a search request.  I wonder if we should have a FilterDefinition and SearchRequest.  A FilterSearchData can reference a filter id or a filter definition + page size / number.  This makes the filter definition re-usable with other methods to add/get filters for example.